### PR TITLE
Set up Tabletop AR project

### PR DESCRIPTION
**Pull request by hud10837**
Related to issue: https://devtopia.esri.com/runtime/kotlin/issues/4438

### Description:

Creates a new `ar` project, with the basic implementation of `TabletopSceneView`, based on the design, as well as a basic microapp.

### Summary of changes:

- 

### Pre-merge Checklist

- a [vTest](https://runtime-kotlin.esri.com/view/all/job/vtest/job/toolkit/) Job for this PR has been run
  - [ ] link:
- Unit and/or integration tests have been added to exercise this PR's logic, and the tests are passing:
  - [ ] Yes
  - [ ] No

### CODE REVIEW

File: toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/TableTopSceneViewScope.kt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+     * Creates a Callout at the specified [geoElement] or the [tapLocation] location on the MapView. The Callout is a composable
+     * that can be used to display additional information about a location on the map. The additional information is
+     * passed as a [content] composable that contains text and/or other content. It has a leader that points to
+     * the location that Callout refers to. The body of the Callout is a rectangular area with curved corners
+     * that contains the [content] lambda provided by the application. A thin border line is drawn around the entire Callout.
+     *
+     * If the given geoelement is a DynamicEntity then the Callout automatically updates its location everytime the
+     * DynamicEntity changes. The content of the Callout however will not be automatically updated.
+     *
+     * Note: Only one Callout can be displayed at a time on the MapView.
```
Comment (gunt0001) [inline toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/TableTopSceneViewScope.kt:60]:
```suggestion
     * Note: Only one Callout can be displayed at a time on the [TableTopSceneView].
```

File: toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/TableTopSceneViewScope.kt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    @Composable
+    public fun Callout(
+        geoElement: GeoElement,
+        modifier: Modifier = Modifier,
+        tapLocation: Point? = null,
+        colorScheme: CalloutColors = CalloutDefaults.colors(),
+        shapes: CalloutShapes = CalloutDefaults.shapes(),
+        content: @Composable BoxScope.() -> Unit
+    ): Unit = sceneViewScope.Callout(geoElement, modifier, tapLocation, colorScheme, shapes, content)
+}
```
Comment (gunt0001) [inline toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/TableTopSceneViewScope.kt:79]: missing newline

File: toolkit/ar/src/test/java/com/arcgismaps/toolkit/ar/ExampleUnitTest.kt [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*
```
Comment (gunt0001) [inline toolkit/ar/src/test/java/com/arcgismaps/toolkit/ar/ExampleUnitTest.kt:1]: I think we can remove this file since we are unlikely going to add non-instrumented tests.

File: toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/Extensions.kt [hunk 4]
```diff
@@ -72,7 +72,9 @@ public fun Activity.launchCustomTabs(pendingSignIn: OAuthUserSignIn?): Unit {
  * @since 200.2.0
  */
 internal fun Activity.launchCustomTabs(authorizeUrl: String, useIncognito: Boolean?) {
-    CustomTabsIntent.Builder().build().apply {
+    CustomTabsIntent.Builder().apply {
+        setShareState(CustomTabsIntent.SHARE_STATE_OFF)
+    }.build().apply {
```
Comment (gunt0001) [inline toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/Extensions.kt:77]: Why this change?

File: toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/OAuthUserSignInActivity.kt [hunk 5]
```diff
@@ -172,6 +177,7 @@ public class OAuthUserSignInActivity : ComponentActivity() {
             }
 
         override fun parseResult(resultCode: Int, intent: Intent?): String? {
+            intent?.data?.toString()?.let { Log.d("CustomTabs", it) }
```
Comment (gunt0001) [inline toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/OAuthUserSignInActivity.kt:180]: Why this change?

File: toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/TableTopSceneView.kt [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    onSingleTapConfirmed: ((SingleTapConfirmedEvent) -> Unit)? = null,
+    onDoubleTap: ((DoubleTapEvent) -> Unit)? = null,
+    onLongPress: ((LongPressEvent) -> Unit)? = null,
+    onTwoPointerTap: ((TwoPointerTapEvent) -> Unit)? = null,
+    onPan: ((PanChangeEvent) -> Unit)? = null,
+    onDrawStatusChanged: ((DrawStatus) -> Unit)? = null,
+    content: (@Composable TableTopSceneViewScope.() -> Unit)? = null
+) {
+    Box(modifier = Modifier.fillMaxSize()) {
+        Icon(
```
[the remark this answers is not in the document]
Comment (hud10837) [reply toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/TableTopSceneView.kt:87]: I used the Box and this Icon to demonstrate that the SceneView renders on top of something. It will be removed in the next PR since the camera feed will be added, I think for this stage of the implementation it makes sense to have it here as it's important that the SceneView has a transparent background

**Comment by hud10837** [comment]
Thanks @gunt0001 I believe I have addressed all of your changes

**Review by gunt0001 — APPROVED** [review]
@hud10837 - looks good 👍
